templates/{{ cookiecutter.metric_slug }}/requirements.txt

```
@@ -0,0 +1,3 @@
+# TODO: fix github to release
+git+https://github.com/huggingface/evaluate.git@metrics-template
+datasets==2.0.0
```

I think this should be enough:
```suggestion
datasets~=2.0
```